src/domains/news/pages/News/News.tsx

```diff
@@ -0,0 +1,51 @@
+import { Breadcrumbs } from "app/components/Breadcrumbs";
+import { Header } from "app/components/Header";
+import { Pagination } from "app/components/Pagination";
+import { NewsCard } from "domains/news/components/NewsCard";
+import React, { useState } from "react";
+
+type Props = {
+	news?: any[];
+};
+
+export const News = ({ news }: Props) => {
+	const [currentPage, setCurrentPage] = useState(1);
+	const crumbs = [
+		{
+			route: "portfolio",
+			label: "Go back to Portfolio",
+		},
+	];
+
+	return (
+		<div data-testid="News" className="flex flex-col min-h-screen -m-5 bg-theme-neutral-100">
```

Comment:
```suggestion
		<div data-testid="News" className="flex flex-col min-h-screen -m-5 bg-theme-neutral-contrast">
```

Same color, but aliased.

Reply:
done @dated